crates/bws/src/cli.rs

```diff
@@ -144,3 +166,12 @@ pub(crate) enum ProjectCommand {
     },
     List,
 }
+
+#[derive(Subcommand, Debug)]
+pub(crate) enum RunCommand {
```

Comment:
I don't think we use this. Should we ?

Reply:
Good catch. I think I missed this when refectoring the command at some point. Removed in f9c9946.